client/spec/package_spec.js

```diff
@@ -40,14 +40,15 @@ describe('Package', function() {
         expect(authoring.getGroupItems('MAIN').count()).toBe(3);
     });
 
-    it('can preview package in a package', function() {
+    xit('can preview package in a package', function() {
```

Comment:
The test should be changed a lot and currently it is using content list and it has to be changed to use monitoring list. SD-3050 task is for changing the tests from using content list to use monitoring list.